charts/osm/crds/multiclusterservice.yaml

```diff
@@ -0,0 +1,64 @@
+# Custom Resource Definition (CRD) for OSM's multicluster config specification.
+# MultiClusterService needs to match the name of the service backing the pods in each cluster
```

Comment:
This comment seems out of place here. I would add this to the Go types that are used for autogenerating the client libs, from which we can autogenerate API docs:
https://github.com/openservicemesh/osm/blob/main/docs/apidocs/README.md

Reply:
Yes, I can add this comment to autogen api as well. @snehachhabria should I remove the comment from here?

Reply:
sure that works